tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -297,6 +297,10 @@ fn time_per_byte_for_atom(a: &mut Allocator, op: &Operator, output: &mut dyn Wri
     linear_regression_of(&samples).expect("linreg failed")
 }
 
+// this function is used for calculating a theoretical cost per node
+// we pass in the time it takes for a byte32 chunk and subtract 4*chunk_time
+// we then divide by two to account for the fact that we are adding a nil atom each time the list grows too
+// this is because atoms are nodes too
```

Comment:
What's the point of this function? It doesn't look like it's used for anything. If you were to make it measure the cost of 1 pair and 1 NIL-atom, you could use the result to validate the model of only charging for the blocks of hashed bytes. But as it is right now, what do you do with the result?

Do you check to see that it's close to 0?